assets/messages.yml

```diff
@@ -173,36 +226,43 @@ messages:
 
         %quick_links%
       fillname: []
-  combat-test:
-    - project: mc
-      name: Combat Test
+  awaiting-response:
+    - project:
+        - bds
+        - mc
+        - mcl
+        - mcd
+        - mce
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Awaiting Response
+      shortcut: ar
       message: |-
-        *Thank you for your report!*
-        However, this issue is {color:#FF5722}*Invalid*{color}.
-
-        We're currently not taking bug reports for the Combat Test experiment; it is for testing the combat system only. Please leave any combat-related feedback on the [reddit post|https://www.reddit.com/r/Minecraft/comments/epy4hv/experimental_combat_snapshot_version_5/] or on the [feedback website|https://aka.ms/JavaCombatSnap].
-
-        If you need help or support you might like to follow a link below.
+        %awaiting_response%
 
         %quick_links%
       fillname: []
-  contact-support:
-    - project: mcpe
-      name: Contact Support
+  combat-test:
+    - project: mc
+      name: Combat Test
+      shortcut: combat
       message: |-
         *Thank you for your report!*
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
-        As this is an issue with your account or billing, you will need to contact [Mojang Customer Support|https://help.minecraft.net/hc/en-us/requests/new] directly.
+        We're currently not taking bug reports for the Combat Test experiment; it is for testing the combat system only. Please leave any combat-related feedback on the [reddit post|https://www.reddit.com/r/Minecraft/comments/epy4hv/experimental_combat_snapshot_version_5/] or on the [feedback website|https://aka.ms/JavaCombatSnap].
 
         %quick_links%
       fillname: []
-  crash-logges-automatically:
+  crash-logged-automatically:
     - project:
         - bds
         - mcpe
         - realms
       name: Crash logged automatically
+      shortcut: crash
```

Comment:
The projects of the separate `crash` shortcuts are mutually exclusive, so it's not an issue. But if that changes, it would become one. I mostly chose `crash` because I couldn't think of any better shortcut.

The messages are all sorted alphabetically. But perhaps it would make sense to merge them into a single `crash` message for it to be less confusing.

Reply:
Changed it to `autocrash`, just to be more explicit.